.github/pull_request_template.md

```diff
@@ -4,4 +4,8 @@
 
 ## Known Issues
 
-## Notes
\ No newline at end of file
+## Notes
+
+## Checklist
+
+- [ ] My name is on the list of contributors (`CONTRIBUTORS.md`) in the pull request source branch.
```

Comment:
whats this?

just a checklist to be sure their name is included in the list of contributors?

Reply:
yes, just to make sure that we have everyone in the contributors list.

It reminds new people to add their name and us to make sure they are listed.

Reply:
thats a good idea!

I can dig it!